docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.post_update.php

```diff
@@ -160,9 +160,9 @@ function su_humsci_profile_post_update_9007() {
 }
 
 /**
- * Disable the gradient hero slider paragraph type on all themes and rows.
+ * Disable the gradient hero slider paragraph type on older themes and older theme rows.
  */
 function su_humsci_profile_post_update_9008() {
-  _humsci_profile_disable_page_paragraph('hs_gradient_hero_slider');
-  _humsci_profile_disable_row_paragraph('hs_gradient_hero_slider');
+  _humsci_profile_disable_page_paragraph('hs_gradient_hero_slider', FALSE);
```

Comment:
Doing this does not re-run this. These lines have already been ran on all production sites and the function `su_humsci_profile_post_update_9008` will never be re-ran. If you need to disable this, you will need a new function.

Reply:
Ok, so we would just rewrite the function with a new number with updating that line?  So like 9010, then the new line with the updated function running?

Reply:
yes

Reply:
Ok, I'll get that updated here and push it up.
Sorry for the confusion.

Reply:
I just pushed up an update for this. Please let me know if it's incorrect.

Reply:
Ok. for anything in the future, it's ideal to wrap the two updates into a single update function.